main.py

```diff
@@ -2,6 +2,50 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    genres = [
+        "Western",
+        "Action",
+        "Dramma"
```

Comment:
The genre name 'Dramma' seems to be a typo, but it is correctly updated to 'Drama' later in the code. Ensure this is intentional.